libafl_bolts/src/llmp.rs

```diff
@@ -1949,9 +1959,7 @@ where
                     ClientId(0),
                     shmem_provider.new_shmem(next_shmem_size(0))?,
                 )],
-                // Broker never cleans up the pages so that new
-                // clients may join at any time
-                keep_pages_forever: true,
+                keep_pages_forever,
```

Comment:
@domenukk any downsides in not keeping the pages in the centralized broker? the main node is the only one reading from the broker

Reply:
yep then don't keep them